package.json

```diff
@@ -1,12 +1,48 @@
 {
-    "name": "mould",
-    "version": "0.1.0",
-    "private": true,
+    "name": "@mouldjs/mould",
+    "version": "0.1.3",
     "scripts": {
         "dev": "node scripts/ensure-test-workdir && cross-env WORKDIR=./test-mould next dev",
-        "build": "next build",
-        "start": "next start"
+        "clean": "rm -rf release/*",
+        "postbuild": "copyfiles -e \"./node_modules/**/*\" -s \"./**/*\" release && rm -rf release/.mould && rm -rf release/.components && rm -rf release/.next",
+        "prebuild": "yarn clean",
+        "build": "tsc -p tsconfig.package.json",
```

Comment:
Before compile TypeScript into JavaScript we probably need to `node scripts/ensure-test-workdir`. Otherwise, [the `.mould` directory](https://github.com/mouldjs/mould/blob/master/runtime/index.tsx#L6) might not exist, which will fail the compilation